crates/executors/src/command.rs

```diff
@@ -132,6 +139,16 @@ impl AgentProfile {
                 "--dangerously-bypass-approvals-and-sandbox",
                 "--skip-git-repo-check",
             ]),
+            mcp_config_path: None,
+        }
+    }
+
+    pub fn qwen() -> Self {
+        Self {
+            label: "qwen".to_string(),
```

Comment:
```suggestion
            label: "qwen-code".to_string(),
```
Clearer if we refer to this as `qwen-code` everywhere.